code/src/OSFramework/Feature/IDirtyMark.ts

```diff
@@ -5,5 +5,6 @@ namespace OSFramework.Feature {
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         getOldValue(rowNumber: number, binding: string): any;
         // clearByRow(row: number): void;
+        saveOriginalValue(rowNumber: number, columnNumber: number): void;
```

Comment:
Please add some comments here and all methods from the Interfaces

Reply:
done